guide/C/ch_txns.xml

```diff
@@ -353,9 +353,9 @@
         <listitem>
           <para>All of the menu items have access keys defined, and these are marked by underlined characters in the
             menu names. Press <keycap>Alt</keycap> + [underlined character] to bring up the menu,
```

Comment:
`[<replacable>underlined character</replacable>]` ?
Can you test which looks nicer?

Reply:
Todo.

Reply:
I think the former is better, because underlined character is not keycap right now.
Forthermore, it is better to change xsl for replaceable to be parsed as italic font.
But I don't have less knowledge about xsl stylesheet.

Reply:
The square brackets serve as our replacement in plain text for Docbook's keycap tag. As we are here in docbook, we should use keycap.

Many former contributors didn't know much more about docbook than "Put your text in a para tag to get a linebreak" resultng in many odd constructions.

On my machine I see it in yelp (xml), firefox (html), and pdf in italic. Where is it not?

Reply:
I'm sorry.
They are shown in italic in my environment.
I just read a wrong file.

Thank you for your confirmation.